python/simpler/task_interface.py

```
@@ -896,6 +900,132 @@ def __repr__(self) -> str:
+    __slots__ = (
+        "name",
+        "members",
+        "buffers",
+        "domain_id",
+        "generation",
+        "mapping_size",
+        "retain_after_run",
+        "_release_fn",
+        "_released",
+        "_freed",
+    )
```

Ruff (0.15.21)

[warning] `GlobalCommDomainHandle.__slots__` is not sorted

Apply a natural sort to `GlobalCommDomainHandle.__slots__`

(RUF023)